.travis.yml

```diff
@@ -67,14 +67,14 @@ matrix:
         #- TESTDOCSEXAMPLES=1
 
     # minimum libcurl we support
-    - python: 2.7
+    - python: 3.8
       env:
         - USECURL=7.19.0-openssl-gssapi
         - USESSL=openssl
         - TESTDOCSEXAMPLES=1
 
     # AVOIDSTDIO tested on one Python version, must be Python 2.x
-    - python: 2.7
+    - python: 3.8
```

Comment:
This configuration should be removed as it does not make sense for python 3.

Reply:
OK configuration removed in next commit. Thx